.github/workflows/lint.yml

```diff
@@ -32,7 +32,6 @@ jobs:
 
       - name: Lint
         run: yarn lint:report
-        continue-on-error: true
```

Comment:
I don't think this is an acceptable solution. We need this to actually upload the linting report to the GitHub annotations.

Reply:
Maybe. But this was working before. What changed?

Reply:
Maybe it wasn't working - e2e always failed due to build errors which may have made this check redundant all along until now? Is there any reason for `yarn lint:report` over `yarn lint` - looking at the implementation yarn lint just dumps everything into stdout.

Reply:
Confirmed it was broken at least since Sep 8 - https://github.com/calcom/cal.com/actions/runs/3370385128/jobs/5592719743